sklearn/ensemble/tests/test_weight_boosting.py

```diff
@@ -113,6 +113,11 @@ def test_iris():
         assert score > 0.9, "Failed with algorithm %s and score = %f" % \
             (alg, score)
 
+        # Check we used multiple estimators
+        assert_true(len(clf.estimators_) > 1)
```

Comment:
Better user `assert_greater(len(clf.estimators_), 1)` to get more informative error messages (when using nosetests).